generator/src/test/java/com/scottlogic/deg/generator/cucumber/ShorterThan.feature

```diff
@@ -116,95 +113,93 @@ Scenario: Running a 'shorterThan' request alongside a non-contradicting inSet co
        | "123"  |
 
 Scenario: Running a 'shorterThan' request alongside a non-contradicting inSet constraint should produce null
-       And foo is shorter than 5
+     Given foo is shorter than 5
        And foo is in set:
        | "12345"  |
        | "123456" |
      Then the following data should be generated:
        | foo    |
        | null   |
 
-Scenario: Running a 'shorterThan' request alongside a null constraint should be successful
-       And foo is shorter than 5
+Scenario: Running a 'shorterThan' request alongside a null constraint should generate null
+     Given foo is shorter than 5
        And foo is null
      Then the following data should be generated:
        | foo  |
        | null |
 
 Scenario: Running a 'shorterThan' request alongside a non-contradicting matchingRegex constraint should be successful
-       And foo is shorter than 3
+     Given foo is shorter than 3
        And foo is matching regex /[☠]{2}/
      Then the following data should be generated:
        | foo    |
        | null   |
        | "☠☠" |
 
-@ignore
-Scenario: Running a 'shorterThan' request alongside a contradicting matchingRegex constraint should produce null
-       And foo is shorter than 2
+Scenario: Running a 'shorterThan' request alongside a contradicting matchingRegex constraint should generate null
+     Given foo is shorter than 2
        And foo is matching regex /[💾]{2}/
```

Comment:
This is an emoji, should we keep the test or remove it as unsupported?

Reply:
The test does pass, It is harmless to keep the test and I think it does provide us with some confidence that we can work with some ASCII character sets even if we don't support them all?

Reply:
ok; it is possible it might fail when running in the generator which emits data to CSV. Only concern is that it implies it is permitted to include this when the documentation says it is not supported.

Reply:
OK, well I can set it to ignore like the other scenarios?

Reply:
Ignore has been added